tests/vibrato_tests.ts

```diff
@@ -105,7 +105,7 @@ function withBend(options: TestOptions, contextBuilder: ContextBuilder): void {
     tabNote({
       positions: [{ str: 2, fret: 10 }],
       duration: 'h',
-    }).addModifier(new Vibrato().setVibratoWidth(120).setHarsh(true), 0),
+    }).addModifier(new Vibrato().setVibratoWidth(120).setVibratoCode(0xeae2), 0),
```

Comment:
To someone reading these tests, it's unclear where the magic codes come from.

Maybe you can use the friendly name (via a local variable `const wiggleVibratoLargeFastest = 0xeae2;`). And add the comment pointing the developer to the relevant SMuFL page for more codes. It's OK if the tests are verbose. They double as a way to teach developers how to use our API.